.github/workflows/reusable_run_notebook.yml

```diff
@@ -44,10 +44,21 @@ jobs:
           name: ${{ inputs.WHEEL_ARTIFACT_NAME }}
           path: wheel
 
-      - name: Install built wheel
+      - name: Install wheel dependencies
+        # First we install the dependencies manually so we can use `--no-index` when installing the wheel.
+        # This needs to be a separate step for some reason or the following step fails
+        # TODO(jleibs): pull these deps from pyproject.toml
         shell: bash
         run: |
-          pip install --find-links wheel rerun-sdk
+          pip install deprecated numpy>=1.23 pillow>=9.5.0 pyarrow==10.0.1 pytest==7.1.2
```

Comment:
This is fragile. Isn't it possible to provide a full path?

```
pip install full/path/to/rerun-sdk-.....whl
```

Reply:
Also, somehow `wheel` is no longer explicitly installed. Not sure if this has an impact.

Reply:
`wheel` is the name of the folder we install from here, not the name of the package, though the overlap in names is confusing. I'll choose a different name.

As for providing an explicit path, it's a bit of a pain in the cross-platform contexts, though here since we're only doing it on linux it would at least be a little easier.